pointcloud_to_grid_node:
  ros__parameters:
    position_x: -5.0
    position_y: 0.0
    cell_size: 0.5
    length_x: 80.0
    length_y: 80.0
    intensity_factor: 0.2
    height_factor: 1.0
    cloud_in_topic: "/left_os1/os1_cloud_node/points"
    mapi_topic_name: "/lidargrid_i"
    maph_topic_name: "/lidargrid_h"
    debug: true